internal/pipe/docker/docker.go

```diff
@@ -262,6 +265,10 @@ func isFileNotFoundError(out string) bool {
 		strings.Contains(out, ": not found")
 }
 
+func isBuildxContextError(out string) bool {
+	return strings.Contains(out, "buildx to switch to context")
```

Comment:
```suggestion
	return strings.Contains(out, "to switch to context")
```

the actual output has a backtick after `buildx`